packages/main/src/plugin/kubernetes/kubernetes-client.spec.ts

```
@@ -259,6 +262,36 @@ class TestKubernetesClient extends KubernetesClient {
+    resolution: 'replace' | 'keep-both',
+  ): { finalContextName: string; finalClusterName: string; finalUserName: string } {
+    return super.resolveNamingConflicts(contextName, sourceContext, existingContext, resolution);
+  }
+  public override addContextToKubeconfig(
+    config: KubeConfig,
+    finalContextName: string,
+    finalClusterName: string,
+    finalUserName: string,
+    sourceContext: KubeContext,
+  ): void {
+    return super.addContextToKubeconfig(config, finalContextName, finalClusterName, finalUserName, sourceContext);
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)